File: 프로그래머스/lv0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.java
class Solution {
    public int solution(int balls, int share) {
        double boonja = 1;
        double boonmo = 1;
		for(int i=0; i<share; i++) {
			boonja *= balls-i;
            boonmo *= share-i;
		}

		
		
		int answer = (int)(boonja/boonmo);
        return answer;
    }
}

File: 프로그래머스/0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.py
def solution(balls, share):
    perm = 1
    count = 0
    for i in range(balls, 0, -1):
        if count == share:
            break
        perm *= i
        count += 1

    facto = 1
    for j in range(share, 0, -1):
        facto *= j

    return int(perm / facto)